ccmlib/cmds/cluster_cmds.py

```diff
@@ -541,6 +544,8 @@ def get_parser(self):
 
     def validate(self, parser, options, args):
         Cmd.validate(self, parser, options, args, load_cluster=True)
+        if self.options.no_wait and self.wait_for_binary_proto:
+            print_("WARN: Both --wait-for-binary-proto and --no-wait were specified. --no-wait will be preferred.")
```

Comment:
Then I'd say the thing to do is have the behavior on by default, but have the flag's default be `False`, so you can detect when the flag is passed, yes?